gemfiles/rails_6.0.gemfile

```diff
@@ -1,8 +0,0 @@
-# This file was generated by Appraisal
-
-source "https://rubygems.org"
-
-gem "rails", "~> 6.0.0"
-gem "activeresource", "~> 5.1.0"
-
-gemspec path: "../"
```

Comment:
why removing 6.0 ?

Reply:
Seemed redundant since we're supporting 6.1 and unnecessary matrix build time. Feel free to add back if you think it's an issue!